Utilities/typography.css

```diff
@@ -3,7 +3,7 @@
   margin: 0px 0px 0.35em;
   font-family: "Roboto", sans-serif;
   font-weight: 300;
-  font-size: 6rem;
+  font-size: 5rem;
   line-height: 1.167;
   letter-spacing: -0.01562em;
```

Comment:
How have you calculated such values for the line-height and letter spacing?

Reply:
did by trial and error & gone through various sites like bootstrap,material UI,chakra